src/pages/ALLO_ManageAllocations.page

```diff
@@ -202,14 +202,12 @@
                                     </div>
                                 </td>
                                 <td>
-                                    <apex:commandButton styleClass="slds-button slds-button_destructive" value="{!$Label.alloDeleteRow}" id="delRowBTN" title="{!$Label.alloDeleteRow}" action="{!delRow}" reRender="theForm" immediate="true">
+                                    <apex:commandButton 
+                                        styleClass="slds-button slds-button_destructive" value="{!$Label.alloDeleteRow}" id="delRowBTN" title="{!$Label.alloDeleteRow}" 
+                                        action="{!delRow}" reRender="theTable,messages" immediate="false">
```

Comment:
Changed the delete button parameters so that it reRenders the table and messages only.  As opposed to the entire form.  Also turned immediate to "false" to prevent what seems like a caching issue on inputField.